src/components/inquiry_artwork_brick/details.tsx

```diff
@@ -0,0 +1,102 @@
+import * as React from "react"
+import * as Relay from "react-relay"
+import styled from "styled-components"
+import colors from "../../assets/colors"
+import TextLink from "../text_link"
+
+export interface DetailsProps extends React.HTMLProps<ArtworkDetails> {
+  artwork: any
```

Comment:
So does that meet that the DetailsProps object should now be empty? It seems like I lose access to some of the props on artwork by doing that.